### PR TITLE
[Hotfix][Fluid] New elements corrections and cpp tests warnings removal

### PR DESCRIPTION
**Pull request by rubenzorrilla**
This PR corrects the boundary terms in both the symbolic and qs_vms elements. Besides I took the chance to remove as warnings as I found aiming to have a clean up for the upcoming release 7.

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element_discontinuous.cpp [hunk 1]
```diff
@@ -131,8 +131,8 @@ void EmbeddedFluidElementDiscontinuous<TBaseElement>::CalculateLocalSystem(
         }
 
         // Recall to swap the boundary term signs because of the interface equilibrium (Neumann) imposition
```
Comment (jcotela) [inline applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element_discontinuous.cpp]: Is this comment still relevant?

**Review by jcotela — DISMISSED** [review]
Thanks for the clean-up

**Comment by jcotela** [comment]
Adding to the release milestone so we don't forget to merge before the freeze.

File: applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element_discontinuous.cpp (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element_discontinuous.cpp]: No. Indeed, it is misleading. Actually, we can even remove the `aux_LHS` and `aux_RHS` arrays.

**Review by jcotela — APPROVED** [review]

**Pull request merged**